test/mesh.jl

```diff
@@ -186,6 +186,12 @@
     topo = topology(mesh)
     @test collect(faces(topo, 2)) == [connect((3,4,1), Triangle)]
     @test collect(faces(topo, 3)) == [connect((1,2,3,4), Tetrahedron)]
+    
+    # test for https://github.com/JuliaGeometry/Meshes.jl/issues/187
+    points = P3[(0,0,0),(1,0,0),(1,1,1),(0,1,0)]
+    tetra  = Tetrahedron(view(points, 4:-1:1))
+    facets = boundary(tetra)
```

Comment:
Please refactor this test so that it is just a simple call to the SimpleMesh constructor. That was the source of the issue. 👌🏽

Reply:
ok

Reply:
@deltaeecs you can keep pushing changes to this branch to update it. You don't need to start a fresh PR from a fresh branch. Let me know if you need some help with that.

Reply:
I change the test file, is it ok?